verify_test.go

```diff
@@ -0,0 +1,378 @@
+package knoxite
+
+import (
+	"errors"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"testing"
+)
+
+var verifyTestCases = []struct {
+	ErrorFunction          func(dir string) error
+	NumberOfExpectedErrors int
+	Percentage             int
+}{
+	{func(dir string) error { return nil }, 0, 100}, // Testing various Percentages
+	{func(dir string) error { return nil }, 0, 70},
+	{func(dir string) error { return nil }, 0, 30},
+	{func(dir string) error { return nil }, 0, 0},
+	{func(dir string) error { return nil }, 0, 256},
```

Comment:
I wonder if the last two cases (0% and 256%) shouldn't actually return errors instead.